### PR TITLE
Monitor template

### PR DESCRIPTION
**Pull request by alstory**
this is a new monitor template that Joe requested. Please review and comment.

### CODE REVIEW

File: monitor-template.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+## Description
+
+Monitors \<what> using the information provided by \<what> which collects metrics from \<what> instances by hitting these endpoints: 
+* <link to item>
+* <link to item>
+
+## Features
+
+For more information on the source data, see <https://...>
```
Comment (barbara-sfx) [inline monitor-template.md]: I'm not sure what this means. Would this follow a feature list and then link out to the originator of the monitor?

Plus the info below doesn't seem like Features to me. Maybe as headings under Usage....

File: monitor-template.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+Monitor Source Code \<link>
+
+Accepts Endpoints: <yes/no>
+
+Multiple Instances Allowed: <yes/no>
+
+
+## Use
```
Comment (barbara-sfx) [inline monitor-template.md]: I'd stick with Usage

File: monitor-template.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+What the monitor is used for -- which metrics and for what -- discussion
```
Comment (barbara-sfx) [inline monitor-template.md]: This sounds like what I would expect in Features, so I'm unclear on the difference between the info in these headings.

File: monitor-template.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+ 
+
+| Name | Type | Default | Description |
+| ---  | ---  | ---    | ---         |
+| `name1` | counter | ✔ | Total connections count per broker |
+| `name2` | gauge | ✔ | Total number of consumers subscribed to destinations on the broker |
+| `name3` | gauge |  | Total number of messages that have been acknowledged from the broker. |
+
+
+### Optional metric configuration
```
Comment (barbara-sfx) [inline monitor-template.md]: Optional ==> Custom   

Refer to slack discussion in Integration Docs for wording to be used here

File: monitor-template.md (hunk 2 above)
Comment (alstory) [reply monitor-template.md]: Done.

File: monitor-template.md (hunk 3 above)
Comment (alstory) [reply monitor-template.md]: Deleted Features heading. Usage is a new heading to get more usage info from the developer. Joe feels like they know more and must be given the space to describe usage.

File: monitor-template.md (hunk 4 above)
Comment (alstory) [reply monitor-template.md]: Done.

**Comment by keitwb** [comment]
This is the kind of stuff that goes in the agent repo, not integrations repo.

File: monitor-template.md (hunk 1 above)
Comment (alstory) [reply monitor-template.md]: Took out the Features heading. The link is to more technical docs in the Integrations repo. It was part of the previous template for a subset of monitors.

**Comment by alstory** [comment]
moved monitor-template file to signalfx-agent repo. will open new pr.

**Pull request closed without merging**